solutionbox/code_free_ml/mltoolbox/code_free_ml/data/trainer/feature_transforms.py

```diff
@@ -64,6 +64,8 @@
 Image.new('RGB', (16, 16)).save(_img_buf, 'jpeg')
 IMAGE_DEFAULT_STRING = base64.urlsafe_b64encode(_img_buf.getvalue())
 
+IMAGE_BOTTLENECK_TENSOR_SIZE = 2048
```

Comment:
There are two types of constants here: one type is part of the public interface, such as schema, transform. Everything else is internal use only. I wonder if we should move the public ones into a different file, so it is easy to point people to the public constant file if they need help creating schema or transforms. The other advantage is that we can keep track of constants that can cause backward compatibility issues.